cmd/virt-launcher/virt-launcher.go

```diff
@@ -108,9 +109,15 @@ func startCmdServer(socketPath string,
 	return done
 }
 
-func createLibvirtConnection() virtcli.Connection {
+func createLibvirtConnection(runWithNonRoot bool) virtcli.Connection {
 	libvirtUri := "qemu:///system"
-	domainConn, err := virtcli.NewConnection(libvirtUri, "", "", 10*time.Second)
+	user := ""
+	if runWithNonRoot == true {
+		user = putil.NonRootUserString
+		libvirtUri = "qemu+unix:///session?socket=/var/run/libvirt/libvirt-sock"
```

Comment:
This would be the case where one libvird instance is running on the host. In that case, we would need for the session mode to use a different path for the socket to avoid accidental connection to the system instance. We can use the same path for root/non-root implementation and ease our findings(in case of debugging) because we are running in the container.

Reply:
Ok, thanks!